### PR TITLE
Calculator fix and refactor

### PR DESCRIPTION
**Pull request by higs4281**
Fixes calculations when user is past full retirement age.  
Our previous requests to SSA were returning benefit values that SSA was bumping to the following year.  
We previously had two methods for requesting benefits: pre-FRA and post-FRA. This unifies to one type of request, one that specifies a claiming month and year. It also simplifies the parsing of results.

## Additions

- New functions for parsing past-FRA responses and calculating values in months.

## Removals
- Code for making a request  and parsing tables in the result. 

## Testing

- CE is testing functionality of the new code. 

## Review

- @marcesher @willbarton

### CODE REVIEW

File: retirement_api/utils/ss_calculator.py [hunk 1]
```diff
@@ -72,6 +73,12 @@ def get_note(note_type, language):
 
 comment = re.compile(r"<!--[\s\S]*?-->")  # regex for parsing indexing data
 
+# calculation constants
+EARLY_PENALTY = 0.00555555  # monthly penalty for months closest to FRA
+EARLIER_PENALTY = 0.004166666  # monthly penalty for earliest claiming ages
+MONTHLY_BONUS = 0.00667  # bonus value for each month's delay past FRA
+ANNUAL_BONUS = 0.08  # annual bonus value for each year's delay past FRA
+
```
Comment (willbarton) [inline retirement_api/utils/ss_calculator.py]: This is just me being curious, no changes needed: are all the magical numbers used in this app published somewhere by SSA or something? How do you figure this out?

Comment (imuchnik) [inline retirement_api/utils/ss_calculator.py]: Well, Magic number is a [code smell](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) and is considered a bad form, so if you would like to refactor, that would be a wise and welcome decision. :smile:

Comment (higs4281) [inline retirement_api/utils/ss_calculator.py]: You could find the values by mining SSA docs, but we relied on SSA and CFPB subject-matter experts who had worked in the mines.

File: retirement_api/utils/ss_calculator.py [hunk 2]
```diff
@@ -234,16 +293,23 @@ def get_retire_data(params, language):
...
                'past_fra': False,
                }
-    BENS = results['data']['benefits']
-    current_age = get_current_age(dobstring)
     results['current_age'] = current_age
+    fra_tuple = get_retirement_age(yobstring)  # returns tuple: (year, momths)
+    if fra_tuple[1]:
+        FRA = "%s and %s months" % (fra_tuple[0], fra_tuple[1])
+    else:
+        FRA = "%s" % fra_tuple[0]
```
Comment (willbarton) [inline retirement_api/utils/ss_calculator.py]: Teensy weensy niggle: could we use `"{}".format()` string formatting for future-friendliness?

**Comment by willbarton** [comment]
Without understanding the mysterious inner workings of social security calculations, the code looks good to me :+1:.

File: retirement_api/utils/ss_calculator.py (hunk 2 above)
Comment (higs4281) [inline retirement_api/utils/ss_calculator.py]: @willbarton fixed.

Comment (willbarton) [inline retirement_api/utils/ss_calculator.py]: :+1:

**Pull request merged**